doc/source/io.rst

```diff
@@ -5,25 +5,22 @@
 .. ipython:: python
    :suppress:
 
-   import os
    import csv
-   from pandas.compat import StringIO, BytesIO
-   import pandas as pd
-   ExcelWriter = pd.ExcelWriter
-
+   import os
+   import matplotlib.pyplot as plt
    import numpy as np
-   np.random.seed(123456)
+   import pandas as pd
+   import pandas.io.date_converters as conv
```

Comment:
I think this should be left where it was. Any reason to move it here?

or even better, I'd use `pd..io.date_converters...`, no extra import is needed, and the code is more explicit